### PR TITLE
OP-1341 Enable Medical Type for Soft Deletion

### PR DESCRIPTION
**Pull request by mohit10696**
See OP-1341.

Initial Documentation added.

### CODE REVIEW

File: doc_user/UserManual.adoc [hunk 1]
```diff
@@ -2650,6 +2650,8 @@ The functions available in this window (similar to other windows in the *_Types_
 * *[.underline]##D##elete*: delete an existing type (cannot be deleted if still used somewhere in the application).
 * *[.underline]##C##lose*: close the window.
 
+Deleted MedicalType will be marked as “Deleted” and will not be shown in the list of Medical Types throught the application. User cannot mark a MedicalType as “Deleted” if it is still used somewhere in the application.
```
Comment (dbmalkovsky) [inline doc_user/UserManual.adoc]:
```suggestion
Deleted Medical Types will be marked as “Deleted” and will not be shown in the list of Medical Types in the application. User cannot mark a Medical Type as “Deleted” if it is still used somewhere in the application.
```
FWIW, `throught` is not a real word.

Comment (mohit10696) [reply doc_user/UserManual.adoc]: Looks like a typo.

File: doc_user/UserManual.adoc [hunk 2]
```diff
@@ -2650,6 +2650,8 @@ The functions available in this window (similar to other windows in the *_Types_
 * *[.underline]##D##elete*: delete an existing type (cannot be deleted if still used somewhere in the application).
 * *[.underline]##C##lose*: close the window.
 
+Deleted Medical Types will be marked as “Deleted” and will not be shown in the list of Medical Types in the application. User cannot mark a Medical Type as “Deleted” if it is still used somewhere in the application.
+
```
Comment (mwithi) [inline doc_user/UserManual.adoc]: I think the note should be harmonized with the above buttons description, this is a suggestion: @dbmalkovsky wdyt?
Please check asciidoc rendering too.

```suggestion
* *[.underline]##D##elete*: delete an existing type. *
* *[.underline]##C##lose*: close the window.

*DELETION - Medical Types cannot be deleted if still used somewhere in the application (i.e. linked to previous data) but can be marked as “Deleted” and will not be shown in the list of Medical Types in the application. If a Medical Type has never been used or is not linked to any previous data, the deletion will be completed with the actual record removal.

```

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: @mwithi your suggested changes sounds good.

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: BTW, why is there no update of image148.png showing the new `Deleted` column?

Comment (mwithi) [reply doc_user/UserManual.adoc]: there is

Comment (dbmalkovsky) [reply doc_user/UserManual.adoc]: sorry; somehow I missed it

**Comment by mohit10696** [comment]
Back in-here. Accepted the changes.

**Pull request merged**